modules/appeals_api/app/models/appeals_api/supplemental_claim.rb

```diff
@@ -176,7 +176,7 @@ def new_evidence_dates
     end
 
     def notice_acknowledgement
-      data_attributes['notice_acknowledgement']
```

Comment:
I'm probably missing something here... Is there a reason we need `? 1 : 'Off'` here? Can't it just be true/false?